jujugui/static/gui/src/app/components/env-list/env-list.js

```diff
@@ -47,6 +47,8 @@ YUI.add('env-list', function() {
         var envName = env.name || env.path;
         envs.push(
           <li className="env-list__environment"
+            role="menuitem"
+            tabIndex="-1"
```

Comment:
Looks like we might want to change this to put it back in the tab order. Not sure if we can get away with that.